{
  "action": {
    "malware": {
      "result": [
        "Infiltrate"
      ],
      "variety": [
        "Ransomware"
      ],
      "vector": [
        "Unknown"
      ]
    }
  },
  "actor": {
    "external": {
      "country": [
        "Unknown"
      ],
      "motive": [
        "Financial"
      ],
      "region": [
        "000000"
      ],
      "variety": [
        "Organized crime"
      ]
    }
  },
  "asset": {
    "assets": [
      {
        "variety": "Unknown"
      }
    ],
    "cloud": [
      "Unknown"
    ]
  },
  "attribute": {
    "availability": {
      "variety": [
        "Obscuration"
      ]
    },
    "confidentiality": {
      "data": [
        {
          "variety": "Unknown"
        }
      ],
      "data_disclosure": "No"
    },
    "integrity": {
      "variety": [
        "Software installation"
      ]
    }
  },
  "confidence": "High",
  "discovery_method": {
    "external": {
      "variety": [
        "Actor disclosure"
      ]
    }
  },
  "incident_id": "4e03a620-6c8d-11e7-8419-7d473c8dcba3",
  "plus": {
    "analysis_status": "Reviewed",
    "analyst": "Breland",
    "created": "2017-10-24T07:11:06.241Z",
    "dbir_year": 2018,
    "event_chain": [
      {
        "action": "mal",
        "actor": "ext",
        "asset": "usr",
        "attribute": "ia"
      }
    ],
    "github": "9748",
    "master_id": "69b5b052-b5b0-4ee2-8682-412282390cf6",
    "modified": "2017-10-26T06:34:34.630Z",
    "sub_source": "phidbr"
  },
  "reference": "http://www.healthcareitnews.com/news/nuance-regains-some-services-after-notpetya-cyberattack",
  "schema_version": "1.3.6",
  "security_incident": "Confirmed",
  "source_id": "vcdb",
  "summary": "Nuance regains some services after NotPetya cyberattack",
  "targeted": "Unknown",
  "timeline": {
    "compromise": {
      "unit": "Seconds"
    },
    "containment": {
      "unit": "Days",
      "value": 8
    },
    "discovery": {
      "unit": "Hours"
    },
    "incident": {
      "month": 7,
      "year": 2017
    }
  },
  "value_chain": {
    "cash-out": {
      "notes": "VERIS 1.3.6: value_chain.development.variety.Cryptocurrency added because action.malware.variety.Ransomware exists.",
      "variety": [
        "Cryptocurrency"
      ]
    }
  },
  "victim": {
    "country": [
      "US"
    ],
    "employee_count": "10001 to 25000",
    "government": [
      "NA"
    ],
    "industry": "541512",
    "region": [
      "019021"
    ],
    "victim_id": "Nuance"
  }
}